agent/config/config.go

```diff
@@ -563,7 +577,11 @@ func (cfg *Config) validateAndOverrideBounds() error {
 	}
 
 	if cfg.DockerStopTimeout < minimumDockerStopTimeout {
-		return fmt.Errorf("Invalid negative DockerStopTimeout: %v", cfg.DockerStopTimeout.String())
+		return fmt.Errorf("Invalid DockerStopTimeout: %v", cfg.DockerStopTimeout.String())
```

Comment:
please follow the convention on error strings from https://github.com/golang/go/wiki/CodeReviewComments#error-strings (here and everywhere else)

Reply:
OK, thanks for the reference, I will revise it accordingly.